app/models/dialog_field_importer.rb

```diff
@@ -28,7 +30,7 @@ def import_field(dialog_field_attributes, export_version = DialogImportService::
       if dialog_field_attributes["type"] == "DialogFieldTagControl"
         set_category_for_tag_control(dialog_field, dialog_field_attributes)
       end
-      dialog_field.save
+      dialog_field.save!
```

Comment:
Why it should be change? I think we can keep it to `save`

Reply:
Because it'd be nice to know when this fails what the error was, and ```save!``` ensures that we know.